apps/react-native-webview-example/app/room/index.tsx

```diff
@@ -1,18 +1,42 @@
-import * as React from "react";
-import { ScrollView, StyleSheet, Text, View } from "react-native";
+import React, { useEffect, useState } from "react";
```

Comment:
this change is not needed. You can do `React.useEffect` and `React.useState` instead. See the other useState uses in this file.

Reply:
👍 will fix that. Thanks for the suggestion.